fandogh_cli/service_commands.py

```diff
@@ -21,8 +21,10 @@ def service():
 @click.option('--port', '-p', 'port', help='The service port that will be exposed on port 80 to worldwide', default=80)
 @click.option('--internal', help='This is an internal service like a DB and the port should '
                                  'not be exposed publicly', default=False, is_flag=True)
+@click.option("--https", help="Enable HTTPS using LetsEncrypt SSL Certificate", default=True, is_flag=True)
+@click.option("--http", help="Enable HTTP and do NOT redirect HTTP traffic to HTTPS", default=True, is_flag=True)
```

Comment:
flag with default value True. how user can disable them?
I think now we it's time to introduce manifest as we have so many options